### ISSUE TITLE
Design and add hover state for all links and buttons

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#2964]
Required by WCAG. Hover state is when mouse hovers over a link before clicking (opposed to focus state which is when a keyboard tabs on the button). Hover states should be visually distinct from focus states

**Comment by Maroyafaied** [comment]
Consider looking into USWDS to incorporate this change.

**Comment by eelzi-ONRR** [comment]
We'd like to use the USWDS default style for all of our buttons, including all the associated states. If this can be applied, it will also resolve issue #2768 

Here's a preview of our buttons using USWDS styling. I used dev tools for this so we can recreate with other pages on the site if we want to look at more buttons before making the change



![Image](https://github.com/DOI-ONRR/onrr.gov-site/assets/115593715/c07e88dc-5ece-400c-b76b-ba55ea84be6b)

**Comment by greensomes** [comment]
The initial changes to the code and SASS have been applied. CMS content needs to be updated with appropriate markup for the USWDS buttons to be used. Here's a running list of content I've updated in preview:
* https://preview-onrr-cms.app.cloud.gov/admin/content/content_blocks/863
*

**Comment by greensomes** [comment]
@eelzi-ONRR, do we have a spreadsheet that lists which pages have buttons on them?

**Comment by eelzi-ONRR** [comment]
I don't think we do. @cthomasONRR that's not something you'd have in your content database, is it?

**Comment by greensomes** [comment]
To address the horizontally aligned buttons for "Oil & Gas" on the reporting landing page, update the following page:

[https://preview-onrr-cms.app.cloud.gov/admin/content/content_blocks/371](https://preview-onrr-cms.app.cloud.gov/admin/content/content_blocks/371)

With this markup:

```
<div class=\"grid-row grid-gap\"><div class=\"grid-col-6\"><a class=\"usa-button\" href=\"/reporting/production\" target=\"_self\">Production</a></div><div class=\"grid-col-6\"><a class=\"usa-button\" href=\"/reporting/revenue\" target=\"_self\">Revenue</a></div></div>
```

**Comment by cthomasONRR** [comment]
Create a new issue for hover state for links.
